### DISCUSSION TITLE
How to Uninstall unwanted things?

### DISCUSSION THREAD

**Discussion by Paradox** [retronas/discussions#81] [Q&A] [answered]
I searched many times, and found nothing, so, sorry if this has already been discussed, but, how do I do the opposite of the "Install things" selection? for example, I installed the gogrepo stuff, but, decided I don't want to use it, without using apt on the command line, how do we reverse the actions taken in the gogrepo playbook, without borking other installed things, of course.

**Comment by sairuk** [comment, marked as answer]
There is currently no way to do this from the UI, we have been discussing options including install/uninstall playbooks, shifting to docker and/or ansible roles none of which, for various reasons ,we've had time to cement down yet.

**Reply by Paradox** [reply]
In the meantime, can someone help me figure out how to remove these things, in the terminal?

**Reply by sairuk** [reply]
Which things do you need to remove just gogrepo?
